homeassistant/components/media_player/squeezebox.py

```diff
@@ -80,17 +80,12 @@ def __init__(self, host, port, username, password):
     def _get_http_port(self):
         """Get http port from media server, it is used to get cover art."""
         http_port = None
-        try:
-            http_port = self.query('pref', 'httpport', '?')
-            if not http_port:
-                _LOGGER.error("Unable to read data from server %s:%s",
-                              self.host, self.port)
-                return
+        http_port = self.query('pref', 'httpport', '?')
+        if http_port:
```

Comment:
Not sure what you are suggesting here ... is it the implicit return None you want to make explicit? Or do you want self.query to be called twice?